examples/aks.preflight.yaml

```diff
@@ -12,7 +12,8 @@ package-sources:
   dir: preflight-packages/
 
 enabled-packages:
-  - "examples.jetstack.io/aks_basic"
+- "examples.jetstack.io/aks_basic"
+- "jetstack.io/pods"
```

Comment:
This is going to fail since the `k8s/pods` is not configured.

Also, shouldn't it be:

```
- id: "examples.jetstack.io/aks_basic"
- id: "jetstack.io/pods"
```
?

Reply:
Ooops, this is a copypasta error... fixed in 8c6c3ecf7ba5dd558b2a74dc5f3fabfcbb38bae0

Reply:
> This is going to fail since the k8s/pods is not configured.
not sure I follow

Reply:
The `k8s/pods` data gatherer is not configured above in the `data-gatherers` section, and this is required for the `pods` package.

Reply:
Ahh, yup. I've opted to remove the pods from both the gke and aks examples in 3f44f1df41324d5f4646c38100b30d58044516fe